packages/astro/src/types/public/config.ts

```diff
@@ -2433,31 +2433,30 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		 * @name experimental.liveContentCollections
 		 * @type {boolean}
 		 * @default `false`
-		 * @version 5.x
+		 * @version 5.10
 		 * @description
 		 * Enables the use of live content collections.
 		 *
 		 */
 		liveContentCollections?: boolean;
 
 		/**
-		 * @name experimental.rawEnvValues
+		 * @name experimental.staticImportMetaEnv
 		 * @type {boolean}
 		 * @default `false`
-		 * @version 5.12
+		 * @version 5.13
 		 * @description
 		 *
-		 * Disables coercion of `import.meta.env` values that are populated from `process.env`, allowing you to use the raw value.
-		 *
-		 * By default, Astro converts your environment variables used through `import.meta.env` in some cases, and this can prevent
-		 * access to some values such as the strings `"true"` (which is converted to a boolean value), and `"1"` (which is converted
-		 * to a number).
+		 * Disables replacement of `import.meta.env` values with `process.env` calls
```

Comment:
This should mention the difference with boolean values if that's what's happening

Reply:
```suggestion
		 * Disables replacement of `import.meta.env` values with `process.env` calls and disables their coercion
```